code-challenges.js

```diff
@@ -16,41 +34,183 @@ const fruit2 = "banana"
 const fruit3 = "cherry"
 const fruit4 = "kiwi"
 
+console.log(stringCompare(fruit1, fruit2))
+
+console.log(stringCompare(fruit3, fruit4))
+
+    // when i ran the code I initially I got 
+
+    // learnacademy@LEARNs-MacBook-Air-5 week-1-assessment-necessities % node code-challenges.js
+    // banana
+    // undefined
+    // cherry
+    // undefined
+
+    // so I had to rewrite my code to return string1 and string2 instead of console.log(string1) or console.log(string1); I'm not sure why this solution did not work
 
+    // const stringCompare = (string1, string2) => {
+    //     if (string1 > string2 ) {
+    //         console.log(string1)
+    //     } else if ( string1 < string2) {
+    //         console.log(string2)
+    //     } else {
+    //         return "Neither string is has more characters"
+    //     }
+    // }
+
+// I'm also going to add these sperators because the output is getting confusing in later problems
+    console.log("--- problem 1 ---")
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
 // Pseudo code:
 
+    // I'll declare a function that takes in one number input and then use a if else statement by comparing the number to the boiling pint number 212 and output whether it is above, below, or at boiling point.
+
+const boilComp = (temp) => {
+    if (temp === 212) {
+        return (`${temp} is at boiling point` )
+    } else if ( temp > 212) {
+        return `${temp} is above boiling point`
+    } else if (temp < 212) {
+        return `${temp} is below boiling point`
+    }
+}
+
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+console.log(boilComp(temp1))
+console.log(boilComp(temp2))
+console.log(boilComp(temp3))
+
+    // learnacademy@LEARNs-MacBook-Air-5 week-1-assessment-necessities % node code-challenges.js
+    // banana
+    // kiwi
+    // undefined
+    // 350 is above boiling point
+    // 212 is at boiling point
+    // learnacademy@LEARNs-MacBook-Air-5 week-1-assessment-necessities % 
+
+    // yup getting an undefined for some reason... lol; finally googled it " randomly getting undefined in my function outputs javascript" - every if else needs a return statement and I didnt have one for a few of the iterations. fixed it and now: 
 
+    //learnacademy@LEARNs-MacBook-Air-5 week-1-assessment-necessities % node code-challenges.js
+    // banana
+    // kiwi
+    // 42 is below boiling point
+    // 350 is above boiling point
+    // 212 is at boiling point
+    // learnacademy@LEARNs-MacBook-Air-5 week-1-assessment-necessities % 
+
+
+    console.log("--- problem 2 ---")
```

Comment:
Awesome work on this problem!  Really great pseudocode and you are showing a great understanding of the concept and how to troubleshoot when getting unexpected outputs.  Great job!